### ISSUE TITLE
Phonebook Delete에 대한 질문

### ISSUE THREAD

**Issue by Da-inKim** [WoonJiLee/project_smcs#15]
운지씨, 안녕하세요!

Phonebook Delete 코드를 보았는데, 질문 드릴 것이 있어서요.

삭제하기위해서 삭제하고자 하는 사람의 이름을 입력받아서 
그 이름을 가진 연락처를 지우는 방식이 맞나요?

**Comment by WoonJiLee** [comment]
네 맞습니다! 원래는 리스트옆에 삭제기능을 넣자고 했지만 뭔가 좀 더 고민해봐야 할거 같아서 일단 사람의 이름을 받아서 삭제하는 방법으로 코딩을 하였습니다.
